src/preact/parse-props.js

```diff
@@ -10,6 +10,10 @@ import * as Preact from '#preact';
 
 import {Slot, createSlot} from './slot';
 
+/** @typedef {*} MediaQueryProps */
+
+/** @typedef {Object<string, AmpElementProp>} AmpElementProps */
```

Comment:
This might be necessary, I don't think `Object` is a generic
```suggestion
/** @typedef {{[key: string]: AmpElementProp}} AmpElementProps */
```

Reply:
Discussed. It is in JSDoc just not in regular .ts